bazel/spec-bundling/esbuild.config-tmpl.mjs

```diff
@@ -26,12 +26,23 @@ async function fetchAndCreateLinkerEsbuildPlugin() {
 // Based on the Bazel action and its substitutions, we run the linker for all inputs.
 const plugins = TMPL_RUN_LINKER ? [await fetchAndCreateLinkerEsbuildPlugin()] : [];
 
+// List of supported features as per ESBuild. See:
+// https://esbuild.github.io/api/#supported.
+const supported = {};
+
+// Async/Await can be downleveled so that ZoneJS can intercept. See:
+// https://github.com/angular/angular-cli/blob/afe9feaa45913/packages/angular_devkit/build_angular/src/builders/browser-esbuild/index.ts#L313-L318.
+if (TMPL_DOWNLEVEL_ASYNC_AWAIT) {
+  supported['async-await'] = false;
```

Comment:
Just to make sure here, if we indicate to downlevel async-await, we mark it as not supported?

Reply:
Correct